db/migrate/20201210173500_create_user_reputation_tokens.rb

```diff
@@ -0,0 +1,24 @@
+class CreateUserReputationTokens < ActiveRecord::Migration[6.1]
+  def change
+    # TODO: Remove
+    drop_table :user_reputation_acquisitions, if_exists: true
```

Comment:
@iHiD Does this need to be removed? In other words, does the exercism.lol environment have this table?

Reply:
Yes, it does. So this line can't be removed. But once this is merged and deployed and run on the server, we can then remove it.